packages/tsx-editor/src/components/Editor.types.ts

```diff
@@ -13,12 +13,12 @@ export interface IEditorProps {
   /** Editor width */
   width: number | string;
 
-  /** Editor code language (default typescript) */
-  language?: string;
-
-  /** Used to get notifications of changes to the editor text */
+  /**
+   * Used to get notifications of changes to the editor text.
+   * This function's identity MUST stay constant to prevent the editor model from being re-created.
```

Comment:
```suggestion
   * This function instance must remain constant to prevent the editor model from being re-created.
```

Reply:
I think I'm going to leave this as-is for now and just merge the PR, because as the next set of work developed, it turns out this prop may not even exist any more... (and if I keep it, I can update the description)